include/jemalloc/internal/extent_externs.h

```diff
@@ -74,4 +74,8 @@ bool extent_merge_wrapper(tsdn_t *tsdn, arena_t *arena,
 
 bool extent_boot(void);
 
+int
+extent_frag_stats(tsdn_t *tsdn, const void *ptr,
```

Comment:
nit: put "int" on same line, use up to 80 chars per line.

re: naming -- maybe, `extent_util_stats_get`? We already use the term utilization in the stats world.